test/with-typescript/index.ts

```diff
@@ -136,13 +447,52 @@ Passwordless.init({
 });
 
 Passwordless.init({
-    createAndSendCustomEmail: async function (input) {},
+    emailDelivery: {
+        service: new STMPService({
+            smtpSettings: {
+                host: "",
+                password: "",
+                port: 465,
+                from: {
+                    name: "",
+                    email: "",
+                },
+            },
+            override: (oI) => {
+                return {
+                    ...oI,
+                    sendRawEmail: async (input) => {
+                        await oI.sendRawEmail(input);
+                    },
+                };
+            },
+        }),
+    },
     contactMethod: "EMAIL",
     flowType: "USER_INPUT_CODE_AND_MAGIC_LINK",
 });
 
 Passwordless.init({
-    createAndSendCustomTextMessage: async function (input) {},
+    smsDelivery: {
+        service: new TwilioService({
+            twilioSettings: {
+                accountSid: "",
+                authToken: "",
+                from: "",
+            },
+            override: (oI) => {
+                return {
+                    ...oI,
+                    sendRawSms: async (input) => {
+                        await oI.sendRawSms(input);
+                    },
+                    getContent: async (input) => {
+                        return await oI.getContent(input);
+                    },
+                };
+            },
+        }),
+    },
     contactMethod: "PHONE",
```

Comment:
Missing:
- thirdpartyemailpassword emaildelivery
- emailpassword emaildelivery
- thirdpaty emaildelivery

Also, you don't need to have one init for each recipe where you are providing a service AND giving it the override object (not of the service, but of the `emailDelivery` / `smsDelivery` object itself.